locales/zh_CN/LC_MESSAGES/vector.po

```diff
@@ -3,178 +3,162 @@
 # This file is distributed under the same license as the taichi package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
 #
-#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: taichi 0.5.14\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-05-12 14:33+0800\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
+"PO-Revision-Date: 2020-05-18 22:29+0800\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.8.0\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: zh_CN\n"
+"X-Generator: Poedit 2.3.1\n"
 
 #: ../../vector.rst:4
 msgid "Vectors"
-msgstr ""
+msgstr "向量"
 
 #: ../../vector.rst:6
 msgid "A vector in Taichi can have two forms:"
-msgstr ""
+msgstr "在Taichi中，向量的表述有两种形式："
 
 #: ../../vector.rst:8
-msgid ""
-"as a temporary local variable. An ``n`` component vector consists of "
-"``n`` scalar values."
-msgstr ""
+msgid "as a temporary local variable. An ``n`` component vector consists of ``n`` scalar values."
+msgstr "作为临时局部变量，一个 ``n`` 分量的向量由标量 ``n`` 组成。"
 
 #: ../../vector.rst:9
-msgid ""
-"as an element of a global tensor. In this case, the tensor is an "
-"N-dimensional array of ``n`` component vectors."
-msgstr ""
+msgid "as an element of a global tensor. In this case, the tensor is an N-dimensional array of ``n`` component vectors."
+msgstr "作为全局张量(global tensor)的构成元素。比如，由向量 ``n`` 组成的N维序列构成了的一个全局张量。"
 
 #: ../../vector.rst:11
 msgid "See :ref:`tensor` for more details."
-msgstr ""
+msgstr "在 :ref:`tensor` 中查看更多细节。"
 
 #: ../../vector.rst:14
 msgid "Declaration"
-msgstr ""
+msgstr "声明向量"
 
 #: ../../vector.rst:17 ../../vector.rst:55
 msgid "As global tensors of vectors"
-msgstr ""
+msgstr "全局张量中的向量"
 
 #: ../../vector.rst
 msgid "Parameters"
 msgstr ""
 
 #: ../../vector.rst:21
 msgid "(scalar) the number of components in the vector"
-msgstr ""
+msgstr "(scalar) 向量中分量数目"
 
 #: ../../vector.rst:22
 msgid "(DataType) data type of the components"
-msgstr ""
+msgstr "(DataType) 向量中分量的数据类型"
 
 #: ../../vector.rst:23
 msgid "(scalar or tuple) shape the tensor of vectors, see :ref:`tensor`"
-msgstr ""
+msgstr "(scalar or tuple)张量的形状(其中的元素就是向量), 在 :ref:`tensor` 中查看更多细节"
```

Comment:
I think it's better to match current cn doc status, `gui.po` for example, translate all type-descriptors in `()`. What do you think? Inputs are welcome!
```suggestion
msgstr "(标量或元组) 张量的形状(其中的元素就是向量), 在 :ref:`tensor` 中查看更多细节"
```

Reply:
This is a good opinion, I will make an amendment as soon as possible